### ISSUE TITLE
Scaleable Power Bar (EG: Warlock Shards)

### ISSUE THREAD

**Issue by Toonic** [teelolws/EditModeExpanded#53]
Would be great if we could the bar separate from the normal UI.

**Comment by teelolws** [comment]
You should be able to. Its Soul Shards in the UI:
![image](https://user-images.githubusercontent.com/49792789/206065385-969d7917-2021-4c98-9551-539f307f8cab.png)
![image](https://user-images.githubusercontent.com/49792789/206065399-443232ab-ac25-47f6-8bad-63a64cb2f4f0.png)
![image](https://user-images.githubusercontent.com/49792789/206065407-c0894bcb-88f5-4673-8acd-2711eff05e26.png)

**Comment by Toonic** [comment]
Yes you can separate them, however you can't scale them up/change the size of them.
